### PR TITLE
Add str method to Rack model

### PR DESCRIPTION
**Pull request by szok**
Added:
- **str** method to Rack
- Filter racks by DataCenter
- Display name and server room in rack list

### CODE REVIEW

File: src/ralph/data_center/models/physical.py [hunk 1]
```diff
@@ -164,6 +164,13 @@ class Rack(NamedMixin.NonUnique, models.Model):
     class Meta:
         unique_together = ('name', 'server_room')
 
+    def __str__(self):
+        return "{} ({}/{})".format(
+            self.name,
+            self.server_room.data_center,
```
Comment (ghost) [inline src/ralph/data_center/models/physical.py:170]: shouldn't  this
`self.server_room.data_center,`
be
`self.server_room.data_center.name,`
?

Comment (szok) [inline src/ralph/data_center/models/physical.py:170]: Beacuse DataCenter **str** return self.name.

**Comment by ghost** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**